### ISSUE TITLE
[$20][MSFT-190] [Member] Contrast color is very low in the banner of "CrowdTesting". 

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2857]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2.  Click on Web Extension tool WAVE for accessibility testing. 
3. Click on contrast button. 
4. Now Observe the banner. 

**Expected Result**
Expected contrast ratio of 4:5:1.

**Actual Result**
Element has insufficient color contrast Ration 2:34:1 (foreground color: #FFFFFF, background color: #25c089)

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.4.1 Use of Color
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=63cedfe6%2D52a5%2D4f33%2D9399%2D00e26b7dd4a2&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%20%2D%20Banner%20Contrast

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096264 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096264 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096264 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
This is related to home page which is out of scope for now.

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097903 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
